test/export/test_serialize.py

```diff
@@ -233,105 +233,114 @@ def forward(self, x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
-                        node2.meta.get("stack_trace", None),
-                    )
+            serialized_artifact = serialize(ep, opset_version={"aten": 0})
+            deserialized_ep = deserialize(serialized_artifact, expected_opset_version={"aten": 0})
+            deserialized_ep.graph.eliminate_dead_code()
 
-                    if node1.target == torch.ops.higher_order.cond:
-                        true_graph1 = getattr(gm1, node1.args[1].target)
-                        true_graph2 = getattr(gm2, node2.args[1].target)
-                        _check_graph_nodes(true_graph1, true_graph2)
-
-                        false_graph1 = getattr(gm1, node1.args[2].target)
-                        false_graph2 = getattr(gm2, node2.args[2].target)
-                        _check_graph_nodes(false_graph1, false_graph2)
-                    elif node1.target == torch.ops.higher_order.map_impl:
-                        map_graph1 = getattr(gm1, node1.args[0].target)
-                        map_graph2 = getattr(gm2, node2.args[0].target)
-                        _check_graph_nodes(map_graph1, map_graph2, False)
-
-                if (
-                    _check_meta and
-                    node1.op not in ("get_attr", "placeholder", "output")
-                ):
-                    # Check "nn_module_stack" metadata
-                    # TODO nn_module_stack is not roundtrippable.
-                    # self.assertEqual(
-                    #     node1.meta.get("nn_module_stack", None),
-                    #     node2.meta.get("nn_module_stack", None),
-                    # )
-                    # Check "source_fn_stack" metadata
-                    self.assertEqual(
-                        node1.meta.get("source_fn_stack", None),
-                        node2.meta.get("source_fn_stack", None),
-                    )
+            orig_outputs = ep.module()(*copy.deepcopy(inputs))
+            loaded_outputs = deserialized_ep.module()(*copy.deepcopy(inputs))
 
-        _check_graph_nodes(ep.graph_module, deserialized_ep.graph_module, _check_meta)
+            flat_orig_outputs = pytree.tree_leaves(orig_outputs)
+            flat_loaded_outputs = pytree.tree_leaves(loaded_outputs)
+
+            for orig, loaded in zip(flat_orig_outputs, flat_loaded_outputs):
+                self.assertEqual(type(orig), type(loaded))
+                if isinstance(orig, torch.Tensor):
+                    if orig.is_meta:
+                        self.assertEqual(orig, loaded)
+                    else:
+                        self.assertTrue(torch.allclose(orig, loaded))
+                else:
+                    self.assertEqual(orig, loaded)
+            self._check_graph_nodes(ep.graph_module, deserialized_ep.graph_module, _check_meta)
+
+        if use_pre_dispatch:
+            _check_graph(pre_dispatch=True)
+            _check_graph(pre_dispatch=False)
+        else:
+            _check_graph(pre_dispatch=False)
```

Comment:
```suggestion
        if use_pre_dispatch:
            _check_graph(pre_dispatch=True)
        _check_graph(pre_dispatch=False)
```

Reply:
yeah -- the `_check_graph(pre_dispatch=False)` is outside of the if statement :P

Reply:
Oh hahaha